libtextsecure/sendmessage.js

```diff
@@ -307,6 +307,13 @@ MessageSender.prototype = {
     const message = new Message(attrs);
     const silent = false;
 
+    // Remove this when we add support for attachments
+    message.attachmentPointers = null;
+    message.preview = null;
+    if (message.quote) {
+      message.quote.attachments = null;
```

Comment:
I believe it would be safer to default to `[]` since there's a lot of `map()` going on without consistent check for truth-ism :)

Reply:
Ah right, good point!